settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'learn'
```

Comment:
I think it's better to have the same name for the project and application

Reply:
Okay, gotcha!